transports/janus_websockets.c

```diff
@@ -293,7 +293,7 @@ static const char *janus_websockets_reason_string(enum lws_callback_reasons reas
 #if (LWS_LIBRARY_VERSION_MAJOR >= 4)
 static lws_retry_bo_t pingpong = { 0 };
 #endif
-
+struct in_addr addr;
```

Comment:
Just as a code style, I'd move this near the `inet_net_pton` calls since those are the only two places where the variable is being used.

Reply:
done